soccer/gameplay/plays/restarts/their_shoot_out.py

```diff
@@ -0,0 +1,149 @@
+import standard_play
+import behavior
+import main
+import robocup
+import play
+import enum
+import skills.move
+import constants
+import evaluation
+
+class TheirShootOut(play.Play):
+	class State(enum.Enum):
+		#where the goalie should start before the game begins
+		starting = 0
+		#prevent the shooting robot from chipping of the robot and shooting a straight shot
+		block = 1
+		#go recieve the ball
+		capture = 2
```

Comment:
This should be the intercept skill rather than capture. It allows a nonzero final velocity to block the ball

Reply:
about to test this also this isn't my most recent change